core/src/test/java/org/mobilitydata/gtfsvalidator/table/AnyTableLoaderTest.java

```diff
@@ -0,0 +1,216 @@
+package org.mobilitydata.gtfsvalidator.table;
+
+import static com.google.common.truth.Truth.assertThat;
+import static org.junit.Assert.*;
```

Comment:
While I'll admit it's not fully consistent, the majority of tests in the project use Truth alone and none of the junit assertions.  I'd argue for sticking to Truth here and using its assertion exclusively, instead of a mix.